src/containers/ArticlePage/components/ConceptsField.tsx

```diff
@@ -80,13 +89,28 @@ const ConceptsField = ({ field, form }: Props) => {
   return (
     <>
       <FieldHeader title={t("form.relatedConcepts.articlesTitle")} />
-      <ElementList
-        elements={concepts}
-        messages={{
-          dragElement: t("form.relatedConcepts.changeOrder"),
-          removeElement: t("form.relatedConcepts.removeArticle"),
-        }}
-        onUpdateElements={onUpdateElements}
+      <DndList
+        items={concepts.map((element, index) => ({ ...element, id: "id" in element ? element.id : index + 1 }))}
```

Comment:
Er det en sjanse for at forklaringer ikke har en ID?

Reply:
Ups, her har jeg fått med noe jeg ikke skulle😅 bra catch 😄